india_banking_connector/connectors/doctype/canara_bank_connector/canara_bank_connector.py

```
@@ -0,0 +1,303 @@
+		if existing_payment_response := self.validate_duplicate_payments(
+			unique_id=unique_id,
+		):
+			return existing_payment_response
+
+		url = self.urls.make_payment
+		headers = self.headers
+
+		signature, payload = self.get_encrypted_payload(method="make_payment")
+		headers["x-signature"] = signature
+
+		response = requests.post(
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)